### ISSUE TITLE
d3.count?

### ISSUE THREAD

**Issue by mbostock** [d3/d3-array#98]
It would count the number of valid numbers in the input.

```js
export default function count(values, valueof) {
  let count = 0;
  if (valueof === undefined) {
    for (let value of values) {
      if (value != null && (value = +value) >= value) {
        ++count;
      }
    }
  } else {
    let index = -1;
    for (let value of values) {
      if ((value = valueof(value, ++index, values)) != null && (value = +value) >= value) {
        ++count;
      }
    }
  }
  return count;
}
```

**Comment by Fil** [comment]
I used valueof=number by defaut, which comes at the (minimal) price of having to keep track of count, but makes a shorter code.

**Comment by mbostock** [comment]
Fixed in #105.
